lib/index.js

```diff
@@ -41,12 +41,21 @@ Server.prototype._createServer = function() {
     return;
   }
 
+  // connections
+  self._connections = {};
+
   var server = net.createServer(function(connection) { //'connection' listener
     new Connection(connection, self, {
       version: self.protocolVersion,
       authKey: self.authKey,
       protocol: self.protocol
     });
+    // push to connections array
+    self._connections[JSON.stringify(connection.address())] = connection;
```

Comment:
I don't think you can use `connection.address()` as the key. Two connections from the same client will clash I think.

You can add an incremental counter in `Server`, and everytime you create a new Connection object, you can have the Connection do something like
```
this.id = this.server.nextConnection++;
```
So it would be something like
```
var reqlConnection = new Connection(connection, self, { ... });
self._connections[reqlConnection.id] = reqlConnection
```
And when you want to close the TCP connection, you can access it via `reqlConnection.connection`